src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -182,4 +198,135 @@ public static int parseHistory(String number) throws ParseException {
         }
         return Integer.parseInt(trimmedIndex);
     }
+
+    /**
+     * Parses {@code String trimmedArgs} into an {@code IdContainsKeywordsPredicate}. Leading and trailing whitespaces
+     * will be trimmed.
+     * @param trimmedArgs keyword that is entered by the user.
+     * @throws ParseException if trimmedArgs is empty or if it contains '/' char.
+     */
```

Comment:
For your JavaDoc comments, would you want to remove the @param line or add in a @return line? Because it seems like the first line of your JavaDoc comment already has the param and return info